device-types/Cisco/N9K-C9372TX-E.yaml

```diff
@@ -0,0 +1,127 @@
+---
+manufacturer: Cisco
+model: Nexus 9372TX-E
+part_number: N9K-C9372TX-E
+slug: n9k-C9372tx-e
+u_height: 1
+is_full_depth: false
+console-ports:
+  - name: console
+    type: rj-45
+power-ports:
```

Comment:
It looks like these power supplies are modular. Is that true? If so, then these should be inside of `module-bays` and the associated modular PSUs should be added to the module-types folder if you can